test/model_simulation/solve_steady_state_problems.jl

```diff
@@ -0,0 +1,65 @@
+### Fetch required packages and reaction networks ###
```

Comment:
Previously all these were commented out. The problem was that they were rather rigorous, at the steady state solving method we tested just are not that good, so they kept erroring (hence we commented it out).

What I did was to add the simple test back in  (since we do allow creating `SteadyStateProblem`s). However, I do not recommend this (and I don't think DiffEq does either) so we could just remove `SteadyStateProblem` functionality all together...